source/_components/media_player.roku.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "Roku"
+title: "Roku Media Player"
 description: "Instructions on how to integrate Roku into Home Assistant."
-date: 2016-05-16 20:0+0000
+date: 2018-10-17 20:0+0000
```

Comment:
Please do not update these, the dates indicate the original creation.

Reply:
Fixed! Thanks!